table_class_optimizer/README.md

```diff
@@ -0,0 +1,137 @@
+# DynamoDB Table Class Optimizer
+## Table of Contents
+- [Overview](#overview)
+- [User Guide](#user-guide)
+  - [Execution](#execution)
+  - [Parameters](#parameters)
+  - [Expected Output](#expected-output)
+  - [Interpretation](#interpretation)
+  - [Best Practices](#best-practices)
+  - [Important Notes](#important-notes)
+- [The Query](#the-query)
+- [FAQ](#faq)
+- [Future Work](#future-work)
+- [Contact Us](#contact-us)
+
+## Overview
+DynamoDB's Standard-IA table class lowers storage costs by 60%, but offsets these savings by increasing throughput costs by 25%. This Athena query on [AWS Cost & Usage Reports](https://docs.aws.amazon.com/cur/latest/userguide/what-is-cur.html) (CUR) data help determine whether switching between Standard and Standard-IA table classes is cost-effective for your DynamoDB tables based on your usage and parameters provided. For more information about table classes and their commonalities, see our [best practice guide](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/CostOptimization_TableClass.html) in the developer documentation.
+
+To get a summary of your total potential savings at the payer level, use the query with **SUMMARY** set in report_type parameter
+To get a detailed cost optimization report on a per-table level, use the query with **DETAILED** set in report_type parameter
+
+Use the full user guide to effectively use the query.
+
+## User guide
+### Execution steps
```

Comment:
Add a new section on how to enable CUR. Be sure to explain the options they should set. Or, refer to documentation and then give the parameters (e.g. granularity required) for the user to input.

Reply:
Will add CUR setup instructions and references